Hackaton2023IOS/Presentation/Common/StoryboardInstantiable/DefaultFileName.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DefaultFileName.swift
+//  Hackaton2023IOS
+//
+//  Created by Kostiantyn Koloskov on 13.05.2023.
+//
+
+import Foundation
+
+protocol DefaultFileName: NSObject {
```

Comment:
Let's consider move it outside StoryboardInstantiable as the are not fully related